app/Http/Controllers/Import/ImportFailureController.php

```
@@ -0,0 +1,514 @@
+                'place' => $request->input('place'),
+                'import_data' => [
+                    'import_id' => $import->id,
+                    'failure_id' => $failure->id,
+                    'created_from_review' => true,
+                    'raw_data' => $failure->raw_data,
+                ],
+            ]);
+
+            // Mark failure as resolved
+            $user = Auth::user();
+            $failure->markAsResolved($user, 'Transaction created from review');
```

PHPStan (2.1.15)

Parameter #1 $reviewer of method App\Models\ImportFailure::markAsResolved() expects App\Models\User, Illuminate\Contracts\Auth\Authenticatable|null given.


(argument.type)